python/lsst/ts/standardscripts/maintel/mtrotator/move_rotator.py

```diff
@@ -0,0 +1,142 @@
+        super().__init__(index=index, descr="Move Rotator")
+
+        self.mtcs = None
+
+        self.rotator_velocity = 3.5  # degrees per second
+        self.short_timeout = 10  # seconds
+        self.long_timeout = 120  # seconds
+
+    @classmethod
+    def get_schema(cls):
+        url = "https://github.com/lsst-ts/"
+        path = (
+            "ts_standardscripts/blob/main/python/lsst/ts/standardscripts/"
+            "maintel/mtrotator/move_rotator.py"
+        )
+        schema_yaml = f"""
+        $schema: http://json-schema.org/draft-07/schema#
+        $id: {url}{path}
+        title: MoveRotator v1
+        description: Configuration for Maintel move rotator SAL Script.
+        type: object
+        properties:
+            angle:
+                description: final angle of the rotator.
+                type: number
+                minimum: -90
+                maximum: 90
+            wait_for_complete:
+                description: >-
+                    whether wait for the rotator to reach the desired angle or
+                    complete the script before the rotator reaches the desired
+                    angle.
+                type: boolean
+                default: true
+        required:
+            - angle
+        additionalProperties: false
+        """
+        schema_dict = yaml.safe_load(schema_yaml)
+
+        base_schema_dict = super().get_schema()
+
+        for properties in base_schema_dict["properties"]:
+            schema_dict["properties"][properties] = base_schema_dict["properties"][
+                properties
+            ]
+
+        return schema_dict
+
+    async def configure(self, config):
+        """
+        Configure the script.
+
+        Parameters
+        ----------
+        config : `dict`
+            Dictionary containing the configuration parameters.
+        """
+        await self.configure_tcs()
+
+        self.target_angle = config.angle
+        self.wait_for_complete = config.wait_for_complete
+
+        await super().configure(config=config)
+
+    async def configure_tcs(self) -> None:
+        """
+        Handle creating MTCS object and waiting for remote to start.
+        """
+        if self.mtcs is None:
+            self.log.debug("Creating MTCS.")
+            self.mtcs = MTCS(
+                domain=self.domain,
+                log=self.log,
+            )
+            await self.mtcs.start_task
+        else:
+            self.log.debug("MTCS already defined, skipping.")
+
+    def set_metadata(self, metadata):
+        """Set the metadata for the script."""
+        metadata.duration = self.long_timeout
+
+    async def run_block(self):
+        """Run the script."""
+        await self.checkpoint(f"Start moving rotator to {self.target_angle} degrees.")
+        await self.mtcs.move_rotator(
+            angle=self.target_angle, wait_for_complete=self.wait_for_complete
+        )
+        if self.wait_for_complete:
```

Comment:
Not sure I understand why you included this checkpoints here. Is the idea to provide a way for people to pause the script? I suggest you change it to be a single message, regardless of the value of `self.wait_for_complete`. Maybe something like:

```
await self.checkpoint(f"Move rotator finished.")
```

Or, if you want to provide information about the `wait_for_complete`, something like this:

```
await self.checkpoint(f"Move rotator finished; wait for complete: {self.wait_for_complete}.")
```